recipe/migrations/abseil_cpp202206230.yaml

```diff
@@ -0,0 +1,20 @@
+__migrator:
+  build_number: 1
+  kind: version
+  migration_number: 2
+
+# TODO: Make sure to remove the associated zip keys in the final file
+# when the migration is complete
+abseil_cpp:
+- '20220623.0'
+# new name, use both for now
+libabseil:
+- '20220623.0'
+# separate output for static builds (depending on C++ version)
+# also need to be migrated
+libabseil_static:
+- '20220623.0'
+grpc_cpp:
+- '1.48'
+
+migrator_ts: 1656693919.0942996
```

Comment:
Yes, I used it explicitly in https://github.com/conda-forge/grpc-cpp-feedstock/pull/196 (with the only exception that I had to also do `use_local: True`); it would also be possible to do this for the other PRs, but there @hmaarrfk had already started modifying the CBC, so I stayed with that. Happy to use the migrator there as well if that's desired.